payloads/library/WiFiCreds/readme.md

```diff
@@ -0,0 +1,25 @@
+# WiFiCreds
+
+* Author: illwill
+* Version: Version 1.0
+* Target: Windows
+
+## Description
+
+Dumps the stored plaintext Wifi SSID & passwords from Windows boxes using 
+Powershell HID attack, then stashes them in /root/udisk/loot/WiFiCreds/
+
+## Configuration
+
+None needed. 
+
+## STATUS
+
+| LED                | Status                                       |
+| ------------------ | -------------------------------------------- |
+| Red (blinking)     | Setting up                                   |

```

Comment:
i guess i could change it to white in the next iteration. also want to add check that theres a result in the loot folder that will be green or red